change/@ni-nimble-components-aca919ed-c567-4f7b-8c65-7310408a45b2.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Out policy for forking templates is to also bring over fast's tests: https://github.com/ni/nimble/blob/874ed79d0d1c0f8228f72de3d12cf562e89d8ada/packages/nimble-components/CONTRIBUTING.md#extend-fast-foundation

- is there a reason it wasn't done?
- I'm now wondering if that was overlooked in other cases we forked templates. Can you double check if that's the case previously? I expect that to not be very time consuming to check and create a tech debt issue of known instances we overlooked copying the tests in. Helps to make the tech debt issue well scoped as if they are too open ended they don't get addressed. Let's discuss if that seems like too much of a burden
- If we do need to bring them in maybe we can keep them in a separate spec file to help with maintenance / diffing against fast. Something like `select.foundation.spec.ts` or something.